tsconfig.json

```diff
@@ -1,24 +1,18 @@
 {
   "compilerOptions": {
-    "incremental": true,                   /* Enable incremental compilation */
```

Comment:
Downside of removing incremental?

Reply:
I had issues compiling the project. Some files that I definitely changed were not rebuilt. I can change it back but for development purposes, I think it is a bit cumbersome.